docs/source/raft_ann_benchmarks.md

```diff
@@ -0,0 +1,169 @@
+# RAFT ANN Benchmarks
+
+This project provides a benchmark program for various ANN search implementations. It's especially suitable for comparing GPU implementations as well as comparing GPU against CPU.
+
+## Installing the benchmarks
+
+You can easily install the benchmarks through conda with the following instructions:
+```bash
+mamba env create --name raft_ann_benchmarks -f conda/environments/bench_ann_cuda-118_arch-x86_64.yaml
+conda activate raft_ann_benchmarks
+
+mamba install -c rapidsai libraft-ann-bench
+```
+The channel `rapidsai` can easily be substituted `rapidsai-nightly` if nightly benchmarks are desired.
+
+Please see the [build instructions](ann_benchmarks_build.md) to build the benchmarks from source.
+
+## Running the benchmarks
+
+### Usage
+There are 4 general steps to running the benchmarks:
+1. Prepare Dataset
+2. Build Index and Search Index
+3. Evaluate Results
+4. Plot Results
+
+### Python-based Scripts
+We provide a collection of lightweight Python based scripts that are wrappers over
+lower level scripts and executables to run our benchmarks. Either Python scripts or
+[low-level scripts and executables](ann_benchmarks_low_level.md) are valid methods to run benchmarks,
+however plots are only provided through our Python scripts.
+#### End-to-end example: Million-scale
+```bash
+# All scripts are present in directory raft/scripts/ann-benchmarks
+
+# (1) prepare dataset
+python scripts/ann-benchmarks/get_dataset.py --name glove-100-angular --normalize
+
+# (2) build and search index
+python scripts/ann-benchmarks/run.py --configuration conf/glove-100-inner.json
+
+# (3) evaluate results
+python scripts/ann-benchmarks/data_export.py --output out.csv --groundtruth data/glove-100-inner/groundtruth.neighbors.ibin result/glove-100-inner/
+
+# (4) plot results
+python scripts/ann-benchmarks/plot.py --result_csv out.csv
+```
+
+#### End-to-end example: Billion-scale
+The above example does not work at Billion-scale because [data preparation](#prep-dataset) is not yet
```

Comment:
Can we be a tad more clear in the wording for this? Rather than "isn't yet supported", let's just say something like "`scripts/get_dataset.py` cannot be used to download the billion-scale datasets because they are so large. You should instead use our [billion-scale datasets guide](...) to download and prepare them." I would highly suggest that we pull the billion-scale data preparation step out into its own file as well so that the user trying to do billion-scale benchmarking (which will likely be often) doesn't have to be exposed to the other lower-level scripts. We can have both the main page and lower-level page link to the billion-scale data prep page.